BAZProjectC1/View/TrendingViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  TrendingViewController.swift
+//  BAZProjectC1
+//
+//
+
+import UIKit
+
+class TrendingViewController: UITableViewController {
+
+    fileprivate var movies: [MovieModel] = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(MovieBasicInfoCell.nib(), forCellReuseIdentifier: MovieBasicInfoCell.identificador)
+    }
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        ApiServiceRequest.getService(urlService: EndpointsList.movieAPI.description, structureType: MovieApiResponseModel.self, handler: {
+            [weak self] dataResponse in
+            if let data = dataResponse as? MovieApiResponseModel {
+                self?.movies = data.results
+                self?.tableView.reloadData()
+            }
+        })
+    }
+
+}
+
+// MARK: - TableView's DataSource
+
+extension TrendingViewController {
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        movies.count
+    }
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell{
```

Comment:
Nitpick: Deja un espacio entre métodos
```suggestion
    }
    
    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell{
```